src/backend/marsha/core/api.py

```diff
@@ -34,6 +35,19 @@
 logger = logging.getLogger(__name__)
 
 
+class NotAllowed(BasePermission):
```

Comment:
You should maybe move this class in an appropriated module ? `utils/permissions.py` ?